ZLocation.Tests/ZLocation.Storage.Tests.ps1

```diff
@@ -3,7 +3,7 @@ Import-Module $PSScriptRoot\..\ZLocation\ZLocation.Storage.psm1 -Force
 Describe 'ZLocation.Storage' {
 
     $originalCount = (Get-ZLocation).Count
-    
+
```

Comment:
Please, keep formatting changes in separate commits or better PRs - it makes reviews much easier.